plugins/authentication/src/pam_password.cpp

```diff
@@ -2,16 +2,14 @@
 
 #include "irods/authentication_plugin_framework.hpp"
 
-#define USE_SSL 1
-#include "irods/sslSockComm.h"
-
 #include "irods/icatHighLevelRoutines.hpp"
 #include "irods/irods_at_scope_exit.hpp"
 #include "irods/irods_auth_constants.hpp"
 #include "irods/irods_client_server_negotiation.hpp"
 #include "irods/irods_logger.hpp"
 #include "irods/irods_pam_auth_object.hpp"
 #include "irods/rcConnect.h"
+#include "irods/sslSockComm.h"
```

Comment:
We may not even need this header anymore, actually

Reply:
Is anything being used from that header?

Reply:
I think the `sslStart` and `sslEnd` calls were the reason for this include. Will confirm that it builds...

Reply:
Removed this include and things still compile, so I think we're good. Resolving.